packages/core/src/components/modal/stories/modal.args.ts

```diff
@@ -160,6 +160,13 @@ export const ModalStoryArgs = {
         category: Category.SLOTS,
       },
     },
+    '--atom-modal-zindex': {
```

Comment:
we need to add it to docs of `atom-steps-modal` too

Reply:
done at [ff7cc0d](https://github.com/juntossomosmais/atomium/pull/617/commits/ff7cc0d21f35c136c5f8de648288ea63ff2e289a)